libs/shared/src/utils/checkIsResponseError.ts

```diff
@@ -0,0 +1,5 @@
+import { IResponseError } from '../types';
```

Comment:
Currently have a PR out for this to merge it separately: https://github.com/novuhq/novu/pull/5338

This file will disappear once that gets merged and we rebase.